index.html

```diff
@@ -9153,6 +9243,175 @@ <h2>Context Definitions</h2>
+      When used in an <a>expanded term definition</a>, its value MUST be a term expanding to <code>@nest</code>.
+      Its value MUST be a <a>string</a>.
+      See <a class="sectionRef" href="#property-nesting"></a> for a further discussion.
+    </dd>
+    <dt><code>@none</code></dt><dd>
+      The <code>@none</code> keyword MAY be aliased and MAY beused as a key in an
+      <a>index map</a>, <a>id map</a>, <a>language map</a>, <a>type map</a>.
+      See <a class="sectionRef" href="#data-indexing"></a>,
+      <a class="sectionRef" href="#language-indexing"></a>,
+      <a class="sectionRef" href="#node-identifier-indexing"></a>,
+      <a class="sectionRef" href="#node-type-indexing"></a>,
+      <a class="sectionRef" href="#named-graph-indexing"></a>, or
+      <a class="sectionRef" href="#named-graph-data-indexing"></a>
+      for a further discussion.</dd>
+    <dt><code>@prefix</code></dt><dd>
+      The <code>@prefix</code> keyword MUST NOT be aliased, and MAY be used as a key in an <a>expanded term definition</a>.
+      Its value MUST be <code>true</code> or <code>false</code>.
+      See <a class="sectionRef" href="#compact-iris"></a>
+      and <a class="sectionRef" href="#context-definitions"></a>
+      for a further discussion.
+    </dd>
+    <dt><code>@reverse</code></dt><dd>
+      The <code>@reverse</code> keyword MAY be aliased and MAY be used as a key in a <a>node object</a>.
+      The unaliased <code>@reverse</code> MAY be used as a key in an <a>expanded term definition</a>.
+      The value of the <code>@reverse</code> key MUST  be an <a>absolute IRI</a>, a <a>relative IRI</a>,
+      or a <a>compact IRI</a> (including <a>blank node identifiers</a>).
+      See <a class="sectionRef" href="#reverse-properties"></a> and
+      <a class="sectionRef" href="#context-definitions"></a> for further discussion.
+    </dd>
+    <dt><code>@set</code></dt><dd>
+      The <code>@set</code> keyword MAY be aliased and MUST be used as a key in a <a>set object</a>.
+      The unaliased <code>@set</code> MAY be used as the value of the <code>@container</code> key within an <a>expanded term definition</a>.
+      Its value MUST be one of the following:
+      <ul>
+        <li><a>string</a>,</li>
+        <li><a>number</a>,</li>
+        <li><a>true</a>,</li>
+        <li><a>false</a>,</li>
+        <li><a>null</a>,</li>
+        <li><a>node object</a>,</li>
+        <li><a>value object</a>, or</li>
+        <li>an <a>array</a> of zero or more of the above possibilities</li>
+      </ul>
+
+      <p>See <a class="sectionRef" href="#sets-and-lists"></a> for further discussion on sets and lists.</p>
+    </dd>
+    <dt><code>@type</code></dt><dd>
+      The <code>@type</code> keyword MAY be aliased and MAY be used as a key in a <a>node object</a> or a <a>value object</a>.
+      The unaliased <code>@type</code> MAY be used as a key in an <a>expanded term definition</a>,
```

Comment:
This makes it sound like an aliased `@type` MAY be used as a key in an expanded term definition... guessing other rules prevent this... probably don't need to change it, but my reading of the previous sentence and this sentence threw me a bit.

Reply:
This is a pattern I came up with, and I'm happy to do something else. The first sentence says that it may be aliased when used in a **node object** or **value object**. The second is that it may be used unaliased as a key in an **expanded term definition**.

Suggestions for a clearer description welcome.